### PR TITLE
Add fqs allergies

### PR DESCRIPTION
**Pull request by luthfib**
Add flag to enable fqs on allergies.

### CODE REVIEW

File: src/fhir/allergies.ts [hunk 1]
```diff
@@ -31,3 +27,78 @@ export function usePatientAllergies() {
     }
   );
 }
+
+const getAllergiesQuery = (upid: string) => gql`
+  {
+    AllergyIntoleranceList(upid: "${upid}") {
```
Comment (pmadria3) [inline src/fhir/allergies.ts:33]: Can we create a query directory and put them there...so something like query/allergies?

File: .changeset/fair-items-accept.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": patch
+---
+
+Add fqs to allergies query.
```
Comment (pmadria3) [inline .changeset/fair-items-accept.md:5]:
```suggestion
Move allergies to utilize FQS. 
```

**Review by pmadria3 — COMMENTED** [review]
Looks good -> just couple comments

**Review by pmadria3 — DISMISSED** [review]
Nice LGTM :D

File: .changeset/fair-items-accept.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": patch
+---
+
+Move allergies to utilize FQS.
```
Comment (luthfib) [inline .changeset/fair-items-accept.md:5]:
```suggestion
Move allergies to utilize FQS.
```

**Review by pmadria3 — DISMISSED** [review]

**Review by mrosata — APPROVED** [review]

**Comment by luthfib** [comment]
DO NOT MERGE UNTIL WE DISCUSS WHAT TO DO FURTHER

File: .changeset/fair-items-accept.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (luthfib) [inline .changeset/fair-items-accept.md:5]:
```suggestion
Add flag to enable FQS on allergies.
```

**Review by mrosata — APPROVED** [review]

**Pull request merged**